src/ui/pages/SidePage/components/IncomingRequest/IncomingRequest.tsx

```diff
@@ -90,6 +94,8 @@ const IncomingRequest = ({ open, setOpenPage }: SidePageContentProps) => {
       await Agent.agent.signifyNotifications.dismissNotification(
         incomingRequest.id
       );
+    } else if (incomingRequest.type === IncomingRequestType.SIGN_DATA_REQUEST) {
+      incomingRequest.peerConnectionEvent?.payload.approvalCallback(false);
```

Comment:
If this is the Dismiss button, I think we should make sure that there is no dismiss button for this type of event. Only accept or cancel. Dismiss implies "do it later" but we have to do it now

Reply:
So is it ok to keep this?

Reply:
No. Remove it from here and remove the dismiss button if there is one.

You should only be able to accept/reject this type of request.

Reply:
Is the "Ignore" button there or gone? I see it in https://github.com/cardano-foundation/cf-identity-wallet/pull/468 videos and not sure where the change is to remove it

Reply:
There was no ignore button